packages/vscode-lit-plugin/package.json

```diff
@@ -36,7 +36,7 @@
 	"dependencies": {
 		"lit-html": "https://github.com/mjbvz/vscode-lit-html.git#1.11.1",
 		"ts-lit-plugin": "1.2.1",
-		"vscode-styled-components": "https://github.com/styled-components/vscode-styled-components#4320c223666ea835e8c63c85573c2704f550b7df",
+		"vscode-styled-components": "https://github.com/styled-components/vscode-styled-components#v1.4.0",
```

Comment:
at some point vscode pushed some breaking change which means master no longer builds for me. some googling shows you need to upgrade whatever depends on vscode, which seems to be this... but we can't upgrade it to the latest version because even with `disable-scripts` it tries to run husky which fails... 🤦‍♂️

Reply:
I know it's very small, but could you separate the build fix into a separate PR?

Reply:
i will, as my other PRs depend on it too. good to separate it